polaris-react/src/components/DataTable/DataTable.scss

```diff
@@ -337,3 +329,19 @@
     box-shadow: var(--p-shadow-base);
   }
 }
+
+.FixedFirstColumn {
+  position: absolute;
+
+  background: white;
```

Comment:
I think we should tie this to `var(--p-surface);`. It's the closest Polaris token that matches this type of use-case I can find. At the very least it ties this `background-color` to the `Card` component's `background-color` which sets the white background for the rest of the table.

Reply:
Done.